src/temaWeek8Exception/main/Repository.java

```diff
@@ -0,0 +1,101 @@
+package temaWeek8Exception.main;
+
+import java.time.Year;
+import java.util.*;
+
+public class Repository {
+	String firstName = "";
+	String lastName = "";
+	String dateOfBirth = "";
+	String gender = "";
+	String ID = "";
```

Comment:
Don't forget about data hiding (private members) and naming conventions (we create identifiers in capital letters only for "constants").